### PR TITLE
[SMALLFIX] a couple of small nits

### PR DESCRIPTION
**Pull request by jsimsa**
- do not sleep before invoking a heartbeat for the first time
- add a generalized utility for printing hex strings

### CODE REVIEW

File: core/common/src/main/java/alluxio/util/FormatUtils.java [hunk 1]
```diff
@@ -72,18 +72,31 @@ public static String byteBufferToString(ByteBuffer buf) {
   }
 
   /**
-   * Parses a byte array into a space separated hex string where each byte is represented in the
-   * format {@code 0x%02X}.
+   * Parses a byte array into a space separates hex string where each byte is represented in the
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/util/FormatUtils.java]: I think it should be `separated` still?

Comment (jsimsa) [inline core/common/src/main/java/alluxio/util/FormatUtils.java]: done

**Comment by calvinjia** [comment]
LGTM

**Comment by jsimsa** [comment]
@calvinjia I suspect the error might be related to the sleeping timer change, I will look into it

**Pull request merged**